jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/migration/MigrationManager.java

```diff
@@ -601,7 +601,10 @@ protected TimerManager getTimerManager(KieSession ksession) {
                                 List<TimerInstance> collected = new ArrayList<>();
                                 for (Long timerId : timers) {
                                     TimerInstance timerInstance = timerManager.getTimerMap().get(timerId);
-
+                                    if(timerInstance==null){
+                                        report.addEntry(Type.WARN, "Could not find any timeer instance to cancel.");
```

Comment:
TimerId may be added also to the report (fix a typo at timer also):
```suggestion
                                        report.addEntry(Type.WARN, "Could not find timer instance with id "+timerId+" to cancel.");
```